src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -50,6 +50,7 @@ values of the `OtlpExporterOptions`
 | `OTEL_EXPORTER_OTLP_ENDPOINT` | `Endpoint`                     |
 | `OTEL_EXPORTER_OTLP_HEADERS`  | `Headers`                      |
 | `OTEL_EXPORTER_OTLP_TIMEOUT`  | `TimeoutMilliseconds`          |
+| `OTEL_EXPORTER_OTLP_PROTOCOL` | `Protocol`          |
```

Comment:
nit: might be good to call out the support values, given we dont support everything from the spec (the http/json)

Reply:
Extended.